### PR TITLE
Add docker

### PR DESCRIPTION
**Pull request by vodkhard**
This fix the #5 issue

## Description
Add Docker to the project to easily install the application

## Motivation and Context
It was very difficult to install the project

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+up:
+	- docker-compose up -d --build
+
+composer:
+	- docker-compose exec php composer install
+
+build: up composer
+	- echo "$(DM_IP)    symfony.local" >> /etc/host
```
Comment (ghost) [inline Makefile]: "S" missing to /etc/host

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    web:
+        image: nginx:latest
+        ports:
+            - "80:80"
+        volumes:
+            - ./:/var/www/symfony
+            - ./docker/nginx/symfony.conf:/etc/nginx/conf.d/symfony.conf
+        depends_on:
+            - php
+    db:
```
Comment (ghost) [inline docker-compose.yml:23]: Can you leave one space here ?

**Review by DrissBelaroussi — DISMISSED** [review]
Cool ! ;)

**Comment by kabaconde** [comment]
All is good !

**Review by ghost — APPROVED** [review]

**Review by ghost — DISMISSED** [review]
It seems to be good !

**Comment by vodkhard** [comment]
> I'm wondering if we really need to use composer here ?

This is a php project so yes I think 😉

**Pull request merged**